app/src/main/cpp/skyline/services/bcat/IBcatService.h

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2020 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <services/serviceman.h>
+
+namespace skyline::service::bcat {
+    /**
+     * @brief IBcatService is used to interact with BCAT
```

Comment:
`used to interact with BCAT (Background Content Asymmetric synchronized delivery and Transmission)`